### ISSUE TITLE
[BUG] 2 Expressions don't work

### ISSUE THREAD

**Issue by OmarOmar93** [SkyCraft78/DiSky#51]
**Name:** `Device Expression and Inviter Expression`

**In-depth description of the bug:**
```
I Just realized that the %discord device of event-member% and the inviter Expression which i found on the bot "Disky" both of them return <none>
```

**Comment by carlodrift** [comment]
Skript version? DiSky version? Server version? Spigot version? Java version? Code used?

**Comment by wallace4BR** [comment]
Can cofirm the issue with discord device of event-member
Skript 2.5.3
Disky 1.10+
Java 11

```
discord command /test:
    trigger:
        reply with "%discord device of event-member%"
```

**Comment by OmarOmar93** [comment]
Disky : 1.11
Skript : latest version
Server : spigot 1.12.1
Java : 11
i didn't use the Inviter code but i found it none in the disky bot in welcome channel

**Comment by WeeskyBDW** [comment]
Propably come from https://github.com/SkyCraft78/DiSky/blob/e63682b443d3d5b2920f295d95f5ae48e82d169f/src/main/java/info/itsthesky/DiSky/skript/expressions/member/ExprDevice.java#L32 method

**Comment by SkyCraft78** [comment]
Fixed in 1.12, thanks for the report ^^
